runtimes/pythonrt/ak_runner/loader.py

```diff
@@ -26,6 +42,12 @@ def name_of(node):
     if isinstance(node, ast.Name):
         return node.id
 
+    if isinstance(node, ast.BinOp):
+        left = name_of(node.left)
+        right = name_of(node.right)
+        op = bin_ops.get(type(node.op), "?")
```

Comment:
shouldn't this be an error?

Reply:
The code that caused it is `(datetime.now() - start).total_seconds()`, not a bug.